### PR TITLE
feat(contrib): improve system related contrib files

### PR DESCRIPTION
**Pull request by zpuskas**
- Make gonic configuration self documenting
- Update systemd files

### CODE REVIEW

File: contrib/config [hunk 1]
```diff
@@ -1,7 +1,52 @@
...
+
+# Interval (in minutes) to check for new music. Default: don't scan
+#scan-interval               0
+#scan-at-start-enabled       false
+#scan-watcher-enabled        false
+#jukebox-enabled             false
+#jukebox-mpv-extra-args      <extra command line arguments to pass to the jukebox mpv daemon>
+
+# A string or character to split genre tags on for multi-genre support (e.g. ;)
+#genre-split                 "\n"
```
Comment (sentriz) [inline contrib/config:52]: i'm not sure that the config file parser does any string unquoting, or if it's even possible to have a new line in the config like that

```suggestion
#genre-split                 ;
```

File: contrib/config [hunk 2]
```diff
@@ -1,7 +1,52 @@
...
+#http-log                    true
+# URL path prefix to use if behind reverse proxy
+#proxy-prefix                <prefix>
+
+# Secure connection settings. Recommended to set up unless Gonic sits behing an
+# SSL enabled reverse proxy server. Disabled by default.
+#tls-cert                    <path to TLS certificatie>
+#tls-key                     <path to TLS private key>
+
+# Gonic's internal state database location
```
Comment (sentriz) [inline contrib/config:23]:
```suggestion
# gonic's internal state database location
```

**Review by sentriz — COMMENTED** [review]
thanks! looks great

File: contrib/config (hunk 2 above)
Comment (zpuskas) [reply contrib/config]: I've actually found the project's name to be incorrectly capitalized in a few more spots in the configuration file, so I fixed all those too.

**Pull request merged**